cmd/libs/go2idl/go-to-protobuf/protobuf/generator.go

```diff
@@ -473,6 +472,7 @@ func protobufTagToField(tag string, field *protoField, m types.Member, t *types.
 	}
 	field.Tag = protoTag
 	// TODO: we are converting a Protobuf type back into an internal type, which is questionable
+	// TODO; Since we are not handling all cases here anyway, maybe we should just remove it?
```

Comment:
I suspect we need a larger refactor of the generator code - to have two type systems (one for protobuf, one for go) and clearly identify which is which.  However, in practice, they are 1-1 and it may not be worth fixing now.  I think the TODO is good, maybe with the more detail or "TODO: allow a way to unambiguously represent a type into two systems at the same time, like Go and Protobuf"